### PR TITLE
Information about the change in Camel 3.12.x

### PR DESCRIPTION
**Pull request by robertfels**
Information about the change in Camel 3.12.x

### CODE REVIEW

**Comment by DavidMitreski5** [comment]
Thank you for your feedback! We’ll look into it and come back to you if we have any questions.

File: docs/ISuite/50-Development/using-camel-simple-expression-language-4688083.md [hunk 1]
```diff
@@ -256,7 +256,8 @@ Incoming message body
 </td>
 <td valign="top">
 
-Incoming message body
+Incoming message body<br/>
+ <i>Deprecated from Camel version 3.12.x</i>
```
Comment (pe-gu) [inline docs/ISuite/50-Development/using-camel-simple-expression-language-4688083.md:260]:
```suggestion
 <i>This expression is deprecated from Apache Camel version 3.12.x.</i>
```

Comment (pe-gu) [reply docs/ISuite/50-Development/using-camel-simple-expression-language-4688083.md:260]: Thank you a lot for this feedback! The documentation will be changed accordingly.

**Review by pe-gu — APPROVED** [review]

**Comment by robertfels** [comment]
https://people.sap.com/robertfels

**Pull request merged**